examples/angular2/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
```

Comment:
:P I just really hate how undepowered `.gitignore` is.

https://github.com/tastejs/todomvc/blob/master/examples/canjs_require/.gitignore#L8-L113